dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraph.java

```diff
@@ -104,9 +104,9 @@ void addPath(DependencyPath path) {
     return versions.get(coordinates);
   }
 
-  // TODO consider whether we need an update class with parent, from, and to, and
+  // TODO consider whether we need an update class with parent, from, and to; and
   // a toString method rather than returning strings
-  List<String> findUpdates() {
+  public List<String> findUpdates() {
```

Comment:
This would benefit from javadoc as to what it returns.  It seems to return a list of human-meaningful strings of updates for the set of configured artifacts in a form that can be dropped into an HTML document (without further encoding? or does freemarker handle that?).

Reply:
done. I don't think this escapes right now. I'm pretty sure the return type is going to change t something more structured as per the TODO comment so I don't want to steel-plate this right now.